src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -185,7 +234,10 @@ private String commandStringGenerator(
       String pathToScript, String scriptName, String... attributes) {
```

Comment:
Good idea, but: if it works, we should focus on the things we haven't finished yet; e.g. add an issue or to-do and leave it for now. Or what do you think?